includes/controls/groups/background.php

```diff
@@ -814,4 +827,20 @@ protected function get_default_options() {
 			'popover' => false,
 		];
 	}
+
+	protected function get_images_sizes() {
+		$sizes = get_intermediate_image_sizes();
+		$sizes[] = 'full';
+		$result = [];
+		foreach ( $sizes as $size ) {
+			$width = get_option( "{$size}_size_w" );
+			$height = get_option( "{$size}_size_h" );
+			if ( ! $width && ! $height ) {
```

Comment:
```suggestion
			if ( ! $width || ! $height ) {
```

Reply:
This is not the logic I need.

Reply:
Are you sure?
Seems like this:
`$result[ $size ] = $size . ' (' . $width . 'x' . $height . ')';`
will result in something like `thumbnail(150xnull)`

Reply:
Yas its also like that with the image size, i looked it up.

Reply:
What does it look like when one of them is empty?

Reply:
Show the Dimensions name insted of showing 

xxxx 450x0